### PR TITLE
Initialize django

### PR DESCRIPTION
**Pull request by bluquar**
Using tutorial here: https://www.digitalocean.com/community/tutorials/build-a-to-do-application-using-django-and-react

To run: python manage.py runserver

Open in browser: http://127.0.0.1:8000/

Screen Shot 2021-11-19 at 5 16 23 PM

### CODE REVIEW

File: backend/backend/settings.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-iya5ffv=5l5^s2ajur!p-#!_zb7opbp18z9oxruz75a#i8*b^d'
```
Comment (lowellbander) [inline backend/backend/settings.py:23]: @bluquar should you put this in an env file or sth?

File: backend/backend/settings.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```
Comment (lowellbander) [inline backend/backend/settings.py:26]: I guess this too?

File: backend/backend/settings.py [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+WSGI_APPLICATION = 'backend.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/3.2/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
```
Comment (lowellbander) [inline backend/backend/settings.py:79]: i've seen lots of folks say they use Postgres, so sth to consider would be using that instead just to be familiar??

**Review by lowellbander — APPROVED** [review]
suh excite! left some comments, but they probs don't need to be addressed now. we could track them as TODOs in this github, or in our Trello? I figure Github would make more sense, wyt?

**Comment by bluquar** [comment]
Yeah I think tracking in GitHub would make sense!

**Pull request merged**